src/lambda/oembed/oembed.js

```diff
@@ -34,7 +34,7 @@ function handler(event, context, callback) {
     );
   }
 
-  const { url, referrer, maxwidth = 900, maxheight = 300 } = params;
+  const { url, referrer, maxwidth = '100%', maxheight = 300 } = params;
```

Comment:
nope, this way we don't pass the embed.ly validator. That's why I couldn't implement `100%` there.

https://embed.ly/providers/validate/oembed?endpoint=https%3A%2F%2Fdeploy-preview-56--testing-playground-com.netlify.app%2Foembed&url=https%3A%2F%2Fdeploy-preview-56--testing-playground-com.netlify.app%2F%3Fmarkup%3DDwIwrgLhD2B2B8AoANtA5gS1o4B6cUc8QA%26query%3DM4YwTgphB2B0DmEAuAhAngJQPYBsIAoByAIwFckktpCBKIA

Reply:
Hmmm OK don't bother this PR then 😓

Reply:
What do you think? Should we report it as a bug on their side?

Reply:
Yeah I think it's a bug indeed

Reply:
Hmm, nope.. A percentage based width isn't allowed according to the spec.

What we can try though, is to omit `width` if a `max_width` isn't set, and add `width: 100%` to the `style` attribute instead.

Reply:
Maybe it's not valid according to the spec, but every browser is handling it well

Reply:
https://twitter.com/meijer_s/status/1266129716475543553?s=19